src/compiler/scala/tools/nsc/transform/SpecializeTypes.scala

```diff
@@ -1744,9 +1744,9 @@ abstract class SpecializeTypes extends InfoTransform with TypingTransformers {
             treeCopy.PackageDef(tree, pid, transformStats(stats ::: specMembers, symbol.moduleClass))
           }
 
-        case Template(parents, self, body) =>
+        case Template(parents, self, tbody) =>
```

Comment:
if less lazy, I would have renamed the ambient `body` element to something more descriptively specific. That allows inner names to be generic, such as body here.